src/App.js

```diff
@@ -1,26 +1,27 @@
 import React from 'react';
-import logo from './logo.svg';
 import './App.css';
+import Comment from "./components/Comment";
+import Message from "./components/Message";
+
+const myComponent = <div>myComponent: Hello World!</div>;
```

Comment:
This isn`t functional component, it`s just variable, please fix it =)

Reply:
Thanks for your notes. I created a new functional component as a new file **FunctionComponent.jsx**.